dbtmetabase/metabase.py

```diff
@@ -593,9 +593,12 @@ def increase_indent(self, flow=False, indentless=False):
                     creator_email = exposure["creator"]["email"]
                     creator_name = exposure["creator"]["common_name"]
                 elif "creator_id" in exposure:
-                    creator = self.api("get", f"/api/user/{exposure['creator_id']}")
-                    creator_email = creator["email"]
-                    creator_name = creator["common_name"]
+                    # If a metabase user is deactivated, the API returns a 404
+                    creator = self.api(
```

Comment:
When critical is set to false, it allows any response (not just 404). Ideally, it should check specifically for 404 that's allowed.